packages/esm-patient-medications-app/src/active-medications/active-medications.component.tsx

```diff
@@ -12,29 +12,35 @@ interface ActiveMedicationsProps {
 const ActiveMedications: React.FC<ActiveMedicationsProps> = ({ patient }) => {
   const { t } = useTranslation();
   const displayText = t('activeMedicationsDisplayText', 'Active medications');
-  const headerTitle = t('activeMedicationsHeaderTitle', 'active medications');
+  const headerTitle = t('activeMedicationsHeaderTitle', 'Active Medications');
 
-  const { data: activePatientOrders, error, isLoading, isValidating } = usePatientOrders(patient?.id, 'ACTIVE');
+  const { data: activePatientOrders, error, isLoading, isValidating } = usePatientOrders(patient?.id);
 
   const launchAddDrugWorkspace = useLaunchWorkspaceRequiringVisit('add-drug-order');
 
   if (isLoading) return <DataTableSkeleton role="progressbar" />;
 
   if (error) return <ErrorState error={error} headerTitle={headerTitle} />;
 
-  if (activePatientOrders?.length) {
+  const activeMedications = activePatientOrders?.filter((order) => {
+    const isActive = !order.dateStopped && order.action !== 'DISCONTINUE';
+    return isActive || (!order.autoExpireDate && order.action !== 'DISCONTINUE');
+  });
```

Comment:
Should this factor in the order's:

- `dateStopped`
- action (should not be `DISCONTINUE`)
- autoExpireDate (should not have elapsed)

And if those are the conditions, should the logic be closer to something like:

```tsx
  const activeMedications = orders?.filter((order) => {
    const currentDate = new Date();
    const hasNotExpired = !order.autoExpireDate || new Date(order.autoExpireDate) > currentDate;
    return !order.dateStopped && order.action !== 'DISCONTINUE' && hasNotExpired;
  });
```

Reply:
This should explain the discrepancy between the number of orders in the active medications table on the Summary page with this PR vs on dev3.

Reply:
Additionally, because we're no longer leveraging the `ACTIVE` parameter to do the filtering on the backend, we should probably handle the filtering in `usePatientOrders` so that consuming components don't need to worry about filtering orders and only deal with rendering the orders they want.